frontend/src/components/input/TextEditor.vue

```diff
@@ -242,6 +245,14 @@ const buttons = computed(() => [
     callback: () =>
       commands.toggleActiveState()(editor.state, editor.view.dispatch),
   },
+  {
+    type: "blockquote",
+    icon: IconBlockquote,
+    ariaLabel: "blockquote",
+    group: "format",
+    isCollapsable: false,
+    callback: () => editor.chain().focus().toggleBlockquote().run(),
+  },
```

Comment:
I'm thinking if the "changing" buttons (like bold, alignment, blockquote) should be separated from the "visibiity" buttons (invisible characters, full height view). But maybe you've aligned with Design about this already?

Reply:
I asked Design. She created a ticket to improve the button arrangement at the end when we added all the additional functionalities.